parser.y

```diff
@@ -2178,44 +2181,42 @@ SplitSyntaxOption:
 		}
 	}
 
-/*******************************************************************************************/
```

Comment:
Should we keep this?

Reply:
The comments without any code in its next line __are ignored__ by goyacc/parser, I can't find them in AST  :joy:

BTW, I think it's acceptable to remove these meaningless comments.